packages/blade/src/components/Alert/Alert.stories.tsx

```diff
@@ -88,11 +90,126 @@ const meta: Meta<AlertProps> = {
   },
 };
 
-const IconButtonTemplate: ComponentStory<typeof AlertComponent> = ({ ...args }) => {
+const AlertTemplate: ComponentStory<typeof AlertComponent> = ({ ...args }) => {
   return <AlertComponent {...args} />;
 };
 
-export const Alert = IconButtonTemplate.bind({});
-Alert.storyName = 'Alert';
+export const Default = AlertTemplate.bind({});
+
+export const HighContrast = AlertTemplate.bind({});
+HighContrast.args = {
+  contrast: 'high',
+  intent: 'notice',
+};
+HighContrast.parameters = {
+  docs: {
+    description: {
+      story: 'A high contrast Alert for more prominent look',
+    },
+  },
+};
+
+export const WithoutActions = AlertTemplate.bind({});
+WithoutActions.args = {
+  actions: undefined,
+};
+WithoutActions.parameters = {
+  docs: {
+    description: {
+      story: 'Alert can also be used without any actions',
+    },
+  },
+};
+
+export const NonDismissable = AlertTemplate.bind({});
+NonDismissable.args = {
+  isDismissable: false,
+};
+NonDismissable.parameters = {
+  docs: {
+    description: {
+      story: 'Alert can be made non dismissable',
+    },
+  },
+};
+
+export const DescriptionOnly = AlertTemplate.bind({});
+DescriptionOnly.args = {
+  description:
+    'The payment was made 6 months ago, therefore you can’t issue refund to this merchant.',
+  intent: 'notice',
+  actions: undefined,
+  title: undefined,
+};
+DescriptionOnly.parameters = {
+  docs: {
+    description: {
+      story: 'Sometimes a description is enough to set the required context',
+    },
+  },
+};
+
+export const PrimaryActionOnly = AlertTemplate.bind({});
+PrimaryActionOnly.args = {
+  description:
+    'There was some internal error while fetching the merchants list, this might also be due to the poor internet connection.',
+  intent: 'negative',
+  actions: {
+    primary: {
+      text: 'Try Refetching',
+      onClick: () => {
+        console.log('Refetch');
+      },
+    },
+  },
+  title: 'Unable to fetch merchants',
+};
+PrimaryActionOnly.parameters = {
+  docs: {
+    description: {
+      story: 'Just a primary action can be enough in some cases',
+    },
+  },
+};
+
+export const FullWidth = AlertTemplate.bind({});
+FullWidth.args = {
+  description: 'Currently you can only accept payments in international currencies using PayPal.',
+  intent: 'notice',
+  actions: undefined,
+  title: undefined,
+  isFullWidth: true,
+};
+FullWidth.parameters = {
+  docs: {
+    description: {
+      story: 'A full width Alert can be used to span the entire width of its container',
+    },
+  },
+};
+
+export const FullBleed: ComponentStory<typeof AlertComponent> = ({ ...args }) => {
```

Comment:
Why are we calling this FullBleed? The story name doesn't seem descriptive enough. How about calling it "Description Only - Borderless"?

Reply:
It's a term designers / printers generally use for edge to edge design. But, right not everyone might be aware, definitely can make it more descriptive.

Reply:
Updated in https://github.com/razorpay/blade/pull/670